js/exporter/jspdf/pdf_table.js

```diff
@@ -34,6 +53,34 @@ export class PdfTable {
                     currentCell.drawTopBorder = false;
                 }
             }
+
+
+            if(!cells[i].skip) {
+                if(!isDefined(cells[i].rect)) {
+                    cells[i].rect = {};
+                }
+
+                if(!isDefined(cells[i].rect.x)) {
```

Comment:
Whether the 'x' member value can be assigned by other code?

Reply:
Currently x,y,w,h may assign in case with band cells